docs/cli-arguments.md

```diff
@@ -37,6 +37,8 @@ Usage of ./nginx-ingress:
     	A ConfigMap resource for customizing NGINX configuration. If a ConfigMap is set,
 	but the Ingress controller is not able to fetch it from Kubernetes API, the Ingress controller will fail to start.
 	Format: <namespace>/<name>
+  -nginx-debug nginx-debug
```

Comment:
seems like the text was copied incorrectly

Reply:
not resovled. `-nginx-debug nginx-debug`. looks incorrect. please double check the output of the `/nginx-ingress -h`